src/client/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const headerElement: JSX.Element = (
```

Comment:
This is not the header I meant, sorry I wasn't clear. `<h1>` is basically means "Heading 1" like in Word or Google Docs. While `<head>` contains metadata of the page, which is not what you want to change with react.